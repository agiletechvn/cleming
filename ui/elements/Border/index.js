import React, { PureComponent } from 'react'
import { View } from 'react-native'

export default class Border extends PureComponent {

  static defaultProps = {
    color: 'rgba(0,0,0,0.5)',
    horizontal: true,
    size: 1,
  }

  render() {
    const { color, size, horizontal, style} = this.props
    let prop1 = 'height', prop2 = 'width'
    if(!horizontal){
     prop1 = 'width'
     prop2 = 'height'
    }
    console.log(prop1,prop2)
    return (
      <View style={{
        ...style,
        overflow: 'hidden',
        [prop1]: size,
        [prop2]: '100%',
      }}>
        <View style={{
          borderStyle:'dotted',
          borderWidth: size,        
          borderColor: color, 
          [prop1]: 0,          
          flex: 1,
        }}/>
      </View>
    )

  }
}

