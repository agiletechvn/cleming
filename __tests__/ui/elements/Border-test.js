import 'react-native';
import React from 'react';
import Border from '../../../ui/elements/Border';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <Border color='rgba(0,0,0,0.5)' size={1} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});