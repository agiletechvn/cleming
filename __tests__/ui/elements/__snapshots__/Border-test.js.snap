// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "space-around",
      "overflow": "hidden",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.5)",
        "borderRadius": 0.5,
        "height": 1,
        "margin": 2,
        "width": 1,
      }
    }
  />
</View>
`;
